src/Shared/Microsoft.DotNet.Scaffolding.Shared/MsBuild/MsBuildProjectContextBuilder.cs

```diff
@@ -20,12 +20,12 @@ public MsBuildProjectContextBuilder(string projectPath, string targetsLocation,
         {
             if (string.IsNullOrEmpty(projectPath))
             {
-                throw new ArgumentNullException(nameof(projectPath));
+                throw new ArgumentException(MessageStrings .ProjectPathNotGiven);
```

Comment:
additional whitespace after "MessageStrings" in `MessageStrings .ProjectPathNotGiven`